packages/steiger/src/features/choose-root-folder/choose-from-guesses.ts

```diff
@@ -0,0 +1,114 @@
+import { sep } from 'node:path'
+import { confirm, isCancel, outro, select } from '@clack/prompts'
+
+import { ExitException } from './exit-exception'
+import { formatCommand } from './format-command'
+import { existsAndIsFolder } from '../../shared/file-system'
+
+const commonRootFolders = ['src', 'app'].map((folder) => `.${sep}${folder}`)
```

Comment:
We could indeed forgo the "universality" and make it search for the FSD root, but the question I have is — how many more cases will we be able to cover by doing so? 

In the issue #40 I've monologued (lol) about the possibility of having plugins provide the logic of searching for the most optimal path, but seeing as a lot of projects are just inside `src` (or `app` in cases of Remix or Next), I really don't think we'd be adding much value by doing all this extra effort. 

If you disagree and think that there are many practical locations where the source may be located, then please provide some examples of paths that this algorithm won't find :)

Reply:
I agree that what we have now most likely covers 90% of cases where a repo contains source code for only one project. But the source folder can not just have a different name than `app` or `src` but also be nested at some level in case of monorepos (it will most likely have a subproject name instead of `src` or `app`) or other specifics.

And yes, you described literally the same thing in the issue, lol, but I guess I missed it and didn't read it

Reply:
Well, 90% is great, in my opinion. After all, the purpose here is not to make a full-fledged terminal UI, but rather make sure we're not abandoning the person blindly when there's an obvious thing they probably meant. 

For the other 10% we simply print the command that will get them what they want and let them take control